www-client/elinks/elinks-0.13_p20180225.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads"
+inherit autotools python-any-r1 vcs-snapshot
+
+COMMIT="0dd3aaabe61fbc5ed0d9a74915678d11d00c2a1c"
```

Comment:
I'd suggest making it `EGIT_COMMIT` for consistency with variables used by git eclass.